features.org

```diff
@@ -205,96 +194,101 @@ More on [[file:manual/Publishing.html][publishing]] (manual)
 
 [[https://orgmode.org/worg/org-tutorials/org-latex-export.html][Tutorial on publishing]] to $\LaTeX$ (Worg)
-
-* Extensibility
+* Take Control of Tasks
   :PROPERTIES:
-  :CUSTOM_ID: extensibiliy
+  :CUSTOM_ID: planning
   :HTML_CONTAINER_CLASS: side-fig
   :END:
 
-#+attr_html: :class main-image :alt Display of some default and custom Org link types
-[[file:resources/img/features/links.svg]]
+#+attr_html: :class main-image :alt Demonstration of planning with org, managing a scheduled todo list
+[[file:resources/img/features/planning.gif]]
 
 ** Description
-Org-mode isn't just capable out of the box, it's
-spectacularly extensible.
 
-Take links as an example. Updating docs for a
-project and find yourself linking to GitHub issues
-a lot? You can add a new link type for the
-document (or globally) where in
-~[[issue:74][version mismatch]]~ the link expands to
-=https://github.com/author/repo/issues/74=.
+Org is an extremely capable task planner, time tracker, and TODO list.
 
-This is just the beginning, you can define
-arbitrary functions to resole links, link to files
-on remote machines with [[https://www.emacswiki.org/emacs/TrampMode][Tramp]], and more.
+Any section can become a /TODO item/ by adding a keyword such as
+=TODO= or =HOLD= to track the state of the task.  You can cycle
+through task states with the shortcuts =S-<left>= and =S-<right>=.
 
-In Org-mode, extensibility isn't an afterthought --- it's baked right into the
-foundations.
+Need add your own set of states?  No problem.  Org lets you customize
+them to support your workflow.
+
+Org /Agenda/ makes it easy to view tasks from multiple files in one
+place.  All you have to do is add one of the TODO keywords.  Org has
+built-in support for task priority, deadlines, scheduled tasks, tags,
+clocking, and more. /Agenda/ can make use of all of it to organize and
+prioritize tasks, seamlessly scaling from a simple TODO list into a
+complete project management suite, all with plain text markup.
+
+*** More
 
-* Time Tracking (Clocking)
+More on [[file:manual/TODO-Items.html][TODO items]] (manual)
+
+Handling [[file:manual/Dates-and-Times.html][dates and times]] (manual)
+* Actionable Agendas
   :PROPERTIES:
-  :CUSTOM_ID: clocking
+  :CUSTOM_ID: agendas
   :HTML_CONTAINER_CLASS: side-fig
   :END:
 
-#+attr_html: :class main-image :alt Using Org to keep track of how long various subtasks took
-[[file:resources/img/features/clocking.svg]]
+#+attr_html: :class main-image :alt An example Org agenda, displaying various todo keywords, priorities, categorisation, and scheduling
+[[file:resources/img/features/agenda.svg]]
 
 ** Description
 
-Keep track of the time you spend on your tasks with Org /Clocking/.
+Plan your day, your way, with ease.
 
-Org provides easy keyboard shortcuts to /clock in/ and /clock out/ of tasks.
+With /agenda views/ --- reports generated from your /TODO items/ ---
```

Comment:
Consider using the ` ---XYZ--- ` style (personal taste).

Reply:
I stuck with the AP style for consistency across all em dashes.